tests/test-files/define-type.txt

```diff
@@ -265,6 +265,25 @@ error: Invalid target for monomorphize attribute
  4 |  (define-type T)
    |  --------------- when parsing define-type
 
+================================================================================
+111 Invalid target for inline attribute
```

Comment:
I don't know how these special test files work, but maybe someone else would know: does the number before the test case title matter at all? This new test starts with the same number (111) as the existing test below, but I do agree that it makes sense to put it here by the monomorphize parser failure test. Maybe we just increment the counter for each of the later tests accordingly?

Reply:
There are no numbers in `parse-declare.txt`. So may be the numbers are optional?